modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -100,6 +102,34 @@ def conditions_related_to_exposure?
         # This will check 'YES' for box 15A.
         # TODO: for disability mapping ticket
       end
+
+      def treatment_centers
+        @pdf_data[:data][:attributes][:claimInformation] = {}
```

Comment:
looking at the [disability PR](https://github.com/department-of-veterans-affairs/vets-api/pull/12725), this will replace what's been done there with an empty object, which will erase everything.

Reply:
yes, and when i merge I will fix it, but for now it needs line 107 to work

Reply:
sounds good. I think the needed PR is approved, so this should be ready for the update.